docs/email-templates.md

```diff
@@ -0,0 +1,43 @@
+# Workshop email templates
+
+## Pre-event reminder
+
+Thank you for signing up for the Jupyter accessibility workshop 
+_How to build inclusive data representations with Frank Elavsky_ on Saturday, January 15 (check the [Jupyter community calendar](https://docs.jupyter.org/en/latest/community/content-community.html#jupyter-community-meetings) to confirm this in your timezone). We're looking forward to saying hello soon!
+
+To join the event, log in to Hopin and join from [the event page](https://hopin.com/events/jupyter-accessibility-workshop-1?code=zQATeMRKrHhGBFQ4SojTUXIdk) or use the link provided in your Hopin ticket email.
+
+Before we meet, here's a few things to remember:
+- If you are interested, you can review [the event agenda](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/events/2022-january-15/group_agenda.md) ahead of time.
+- By participating in the event, you are agreeing to follow the [event Code of Conduct](https://github.com/Quansight-Labs/jupyter-accessibility-workshops/blob/main/code-of-conduct.md).
+- The main session/speaker will be recorded and publicly available post-event.
+
+If you have any questions, reach out via ???
```

Comment:
I don't know where to link to. Twitter? My email?

Reply:
I suggest email

Reply:
Done!